creator/organizations/mutations.py

```diff
@@ -113,10 +118,146 @@ def mutate(self, info, id, input):
+        )
+        user = graphene.ID(
+            required=True,
+            description="The ID of the user to add the to the organization",
+        )
+
+    organization = graphene.Field(OrganizationNode)
+
+    def mutate(self, info, organization, user):
+        """
+        Add a member to an organization
+        """
+        user_id = user
+        user = info.context.user
+        if not user.has_perm("organizations.change_organization"):
+            raise GraphQLError("Not allowed")
+
+        try:
+            _, organization_id = from_global_id(organization)
+            organization = Organization.objects.get(pk=organization_id)
+        except Organization.DoesNotExist:
+            raise GraphQLError(
+                f"Organization {organization_id} does not exist"
+            )
+
+        try:
+            _, user_id = from_global_id(user_id)
+            member = User.objects.get(id=int(user_id))
+        except (User.DoesNotExist, TypeError):
+            raise GraphQLError(f"User {user_id} does not exist.")
+
+        if not member.organizations.filter(pk=organization_id).exists():
+            organization.members.add(member)
+            message = (
+                f"{user.display_name} added {member.display_name} "
+                f"to organization {organization.name}"
+            )
+            event = Event(
+                organization=organization,
+                description=message,
+                event_type="MB_ADD",
+            )
+        else:
+            raise GraphQLError(
+                f"User {member.display_name} is already a member of "
+                f"organization {organization.name}."
+            )
+
+        organization.save()
+
+        if not user._state.adding:
+            event.user = user
+        event.save()
+
+        return AddMemberMutation(organization=organization)
+
+
+class RemoveMemberMutation(graphene.Mutation):
+    """ Mutation to remove a member from an organization """
+
+    class Arguments:
+        """ Arguments to remove a member from an organization """
+
+        organization = graphene.ID(
+            required=True, description="The ID of the organization to update"
+        )
+        user = graphene.ID(
+            required=True,
+            description="The ID of the user to remove the to the organization",
+        )
+
+    organization = graphene.Field(OrganizationNode)
+
+    def mutate(self, info, organization, user):
+        """
+        Remove a member from an organization
+        """
+        user_id = user
+        user = info.context.user
+        if not user.has_perm("organizations.change_organization"):
+            raise GraphQLError("Not allowed")
+
+        try:
+            _, organization_id = from_global_id(organization)
+            organization = Organization.objects.get(pk=organization_id)
+        except Organization.DoesNotExist:
+            raise GraphQLError(
+                f"Organization {organization_id} does not exist"
```

Comment:
```suggestion
                f"Organization {organization_id} does not exist."
```
i'm sorry the other ones had a period it bugged me 😝